paasta_tools/contrib/rightsizer_soaconfigs_update.py

```diff
@@ -276,7 +279,7 @@ def get_recommendations_by_service_file(
             result["cluster"],
         )  # e.g. (foo, marathon-norcal-stagef)
         instance_type = result["cluster"].split("-", 1)[0]
-        rec: Union[KubernetesRecommendation, CassandraRecommendation]
+        rec: Union[KubernetesRecommendation, CassandraRecommendation] = {}
```

Comment:
not super necessary, but pyright was correctly calling this out as a potentially unbound variable below (and I figured this is innocuous enough to sneak in here :p)